samples/nrf9160/lwm2m_client/overlay-queue.conf

```diff
@@ -26,5 +26,7 @@ CONFIG_LTE_PSM_REQ_RAT="00001111"
 CONFIG_LTE_EDRX_REQ=y
 # Request 10.24 eDRX
 CONFIG_LTE_EDRX_REQ_VALUE_LTE_M="0001"
+CONFIG_LTE_EDRX_REQ_VALUE_NBIOT="0001"
```

Comment:
```suggestion
CONFIG_LTE_EDRX_REQ_VALUE_NBIOT="0010"
```

Reply:
Values "0000" and "0001" are applicable only in WB-S1 mode and interpreted as "0010" in NB-S1